.gitignore

```diff
@@ -0,0 +1,2 @@
+**/node_modules/
+package-lock.json
```

Comment:
package-lock is an important file, it will handle dependencies and versions for you when you do npm install, is a best practice to do not ignore package-lock unless specific purpose.
This file will help your team have a lock functional version for your app